### DISCUSSION TITLE
webGIS Erfassen Autocomplete

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#224] [General]
Hallo Jürgen,
mit Build 6.24.4402 funktionieren beim Editing die Autocomplete-Felder nicht mehr. 
An der Konfiguration kann es nicht liegen, außer es hat sich was geändert zu der in dieser discussion https://github.com/gis-eni/webgis-forum/discussions/46# dokumentierten.
Ich habe die gleiche cms-Konfiguration mit Build 5.24.1101 probiert - da geht es noch.
Ab wann es nicht mehr funktioniert, kann ich leider nicht nachvollziehen .

Hier noch die Fehlermeldung im Entwickler-Tool:
![image](https://github.com/user-attachments/assets/c890e5dd-47bc-43c9-8dbf-a4ab03ec33f0)

lg

**Comment by gis-eni** [comment]
Der oben gezeigte Fehler ist die Folge eines vorgänger Fehlers. Wenn du ein deinem Autocomplete etwas eingibst, werden Request zum Server geschickt. Da sollte in der Regel ein Array von Werten zurück kommen, wenn dort aber schon ein Fehler zurück kommt, werden natürlich auch keine Werte angezeigt. Bekommst du da Fehler (schau in der Konsole bei Network mit):

![image](https://github.com/user-attachments/assets/24cb30cf-7c57-4381-bac6-540bc16e1341)

Und wenn ja, wie lautet der Fehler da bei dir?

**Comment by marko-jer** [comment]
Die Fehlermeldung aus der Konsole:
![image](https://github.com/user-attachments/assets/3761b6d4-5754-4fdb-806b-873081ea57c5)
Ich hoffe es hilft dir weiter.

Die gleiche Konfiguration (gleiches CMS) funktioniert einwandfrei mit Build 5.24.1101.

**Reply by gis-eni** [reply]
Was ist die Datenquelle für deine Auswahlliste? Greifst du direkt auf eine Datenbank zu oder geht es über eine DataLinq Abfrage?

**Reply by marko-jer** [reply]
die Datenquelle für die Auswahlliste ist die Datenbank:
im CMS schaut es dann so aus
![image](https://github.com/user-attachments/assets/21f510b3-5d08-4c79-8912-d48191c26e80)

**Reply by gis-eni** [reply]
Ok, danke für die Info. Konnte ich nachvollziehen.
Anscheinend hat sich was in der zugrunde liegenden SqlClient Library von Microsoft getan, von unserer Seite hat sich nix geändert. Konnte das Problem trotzdem fixen (und es ist wenn, "nur" beim Editing-Autocomplete aufgetreten)
Build 6.24.5001 liegt bereits am Server.

**Reply by marko-jer** [reply]
THX!
